### PR TITLE
buyers guide homepage

### LINKED ISSUE #1820: Buyers Guide: home page
Based on design in https://github.com/mozilla/foundation.mozilla.org/issues/1739

WIP comps https://redpen.io/p/qz6bdb2811e058ffb1

Filter svg icon [icon-filter.svg.zip](https://github.com/mozilla/foundation.mozilla.org/files/2448495/icon-filter.svg.zip)

Hero background image

- [x] Intro text and banner

- [x] Animated smiley face that changes as products scroll correlating the state of the face to the level of the products' creepiness (products are grouped into 5 creepiness categories) 

- [x] Animated face is sticky at the top of browser

- [x] Products are ordered according to level of creepiness (least creepy to most creepy)

- [x] Products display their aggregated score of likelihood to buy by users (thumbs up for likely to buy or thumbs down for unlikely to buy)

- [ ] Filter 'bubble' that expands (animation opportunity!) upon click into a larger bubble containing filtering options based on 'creepiness' and 'likelihood to buy' example: https://teatrlalka.pl/en/kalendarz

- [ ] User is able to filter products according to aggregated scores of creepiness (creep-o-meter slider with range) and likelihood to buy (radio buttons)

- [ ] Empty state for filters shows text and reset link

- [x] User can click on a product to go to that product's detail page

### Copy
Shop Safe This Holiday Season (Zilla Slab, Light, 56/64, center aligned - mobile 36/40)

Teddy bears that connect to the internet. Smart speakers that listen to commands. Great gifts—unless they spy on you. We created this guide to help you buy safe, secure products this holiday season. (.body-large, center aligned)

Filter products below by user ratings on our Creep-O-Meter. Click on a product to rate it. (Zilla Slab, Italic, 18/24, #9B9B9B)

### PR DESCRIPTION
**Pull request by Pomax**
Closes #1820 

Does not implement the filter at the moment, which should be done in follow-up.

### CODE REVIEW

**Comment by alanmoo** [comment]
Assigning to @kristinashu for design review, but in the interest of time zones I'll try and get to this in a little bit for code review first.

File: network-api/networkapi/buyersguide/templates/bg_base.html [hunk 1]
```diff
@@ -28,7 +28,8 @@
         </div>
       </div>
     </header>
-    <div class="primary-nav">
+
+    <div class="primary-nav sticky-top bg-white">
```
Comment (alanmoo) [inline network-api/networkapi/buyersguide/templates/bg_base.html]: Add webkit sticky to the CSS for this bit manually?

File: source/sass/buyers-guide/views/homepage.scss [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    box-sizing: content-box;
+    margin: 1px;
+    overflow: hidden;
+  }
+
+  .creepiness-slider {
+    z-index: 0;
+    top: 40px;
+    padding-top: 20px;
+
```
Comment (alanmoo) [inline source/sass/buyers-guide/views/homepage.scss]: Let's remove the `sticky-top` class and just add the following:
```suggestion
position: sticky;
position: -webkit-sticky;

```

File: network-api/networkapi/buyersguide/templates/buyersguide_home.html [hunk 3]
```diff
@@ -1,15 +1,49 @@
...
 
 {% block guts %}
+<div class="container-fluid text-center page-header bg-gray">
+  <!-- home banner as css background -->
+</div>
 
 <div class="container">
-  <h1>Buyer's guide homepage!</h1>
+    <div class="row">
+      <div class="product-listing bg-white p-5 col-12 col-md-10 offset-md-1">
```
[the remark this answers is not in the document]
Comment (kristinashu) [reply network-api/networkapi/buyersguide/templates/buyersguide_home.html]: For tablet and mobile, cards should fill the width:

image

File: network-api/networkapi/buyersguide/templates/bg_base.html (hunk 1 above)
Comment (Pomax) [reply network-api/networkapi/buyersguide/templates/bg_base.html]: going with @mmmavis's recommendation!

**Comment by kristinashu** [comment]
On desktop the face scroll is perfect! 

I will approve design but a few notes for followup:
- mobile: face should be sticky at the top (like desktop) https://redpen.io/rab5c2beee4e128e63
  - tested on my iphone in safrari
- missing the text "Filter products below by user ratings on our Creep-O-Meter. Click on a product to rate it." in Zilla Slab, Italic, 18/24, #9B9B9B https://redpen.io/gva8805a4647c1aba9
- as mentioned in Alan's comment, at narrower widths, the product thumbnails should take up more space

**Comment by Pomax** [comment]
@kristinashu I assumed the filter text was only relevant once we actually have filtering, though, so I left it out. Does it make sense to include it if there's not filtering option?

As for 1, this is a missing css prefix so should be fixed with the CSS fix alan suggested too.

**Comment by kristinashu** [comment]
Ha Pomax! You're completely right, ignore that text.

**Comment by Pomax** [comment]
@kristinashu one more pass? https://foundation-mofostaging-pr-1942.herokuapp.com/privacynotincluded/

**Comment by kristinashu** [comment]
#### Yeah! Thumbnails at narrow widths look much better!

#### But I think the padding in the intro text looked better before.

Screen cap now:
image


Redpen comp:
image

Here's the grid if that helps:
image

#### Sorry I probably missed this before but the banner height seems tall. It should be reduced to 304px.
image

#### None of these are blockers tho!

**Pull request merged**